src/views/ReadArticle.js

```diff
@@ -200,10 +210,11 @@ export class Article extends Component {
         readTime,
         comments
       } = retrievedArticle);
-      ({ username, firstName, lastName, image } = author);
+      ({ username, firstName, following, lastName, image } = author);
     }
     const isAuthor = isCurrentUserAuthor(username, currentUser);
-
+    const { following: followObject } = this.props;
+    if (followObject.status !== null) following = followObject.status;
```

Comment:
What happens if status === undefined? Can't you just check for `if(followObject.status)`?

Reply:
I see, but what if I check for `if(followObject.status)` and status becomes `false` since it is a boolean?